server/api/reports.py

```diff
@@ -1082,7 +1090,7 @@ def sampled_batch_rows(election: Election, jurisdiction: Jurisdiction = None):
                         sampler_contest.from_db_contest(contest),
                         0,
                     )
-                    if is_audited and audit_results and not is_combined
+                    if audit_results
```

Comment:
Unrelated to the change, but adding a test caught the fact that `is_combined` isn't actually defined on this line. It was a copy/paste error that never manifested due to short-circuiting (I assume). Any reasonable type-checking/linting should have caught this, so I'm a bit annoyed

Reply:
Dang good catch - I've been wondering whether some version upgrades would give us better Python typechecking